samples/core/src/test/java/io/oasp/gastronomy/restaurant/staffmanagement/batch/impl/staffimport/StaffImportConfig.java

```diff
@@ -0,0 +1,151 @@
+package io.oasp.gastronomy.restaurant.staffmanagement.batch.impl.staffimport;
```

Comment:
Das ist jetzt ein Batch in JavaConfig. Ich finde es unschön, dass dauerhaft zu mischen. Wenn die Config unten funktioniert können, wir wegen mir komplett auf Java-Config umstellen. Bitte erstelle dazu ein eigenes Ticket, in dem wir das ggf. nochmal kurz diskutieren.